--- ldns-1.7.0/ax_python_devel.m4.orig	2017-04-17 10:21:17.538303304 +0200
+++ ldns-1.7.0/ax_python_devel.m4	2017-04-17 10:27:08.141632634 +0200
@@ -227,7 +227,7 @@
 		then
 			# use the official shared library
 			ac_python_library=`echo "$ac_python_library" | sed "s/^lib//"`
-			PYTHON_LIBS="-L$ac_python_libdir -l$ac_python_library"
+			PYTHON_LIBS="-l$ac_python_library"
 		else
 			# old way: use libpython from python_configdir
 			ac_python_libdir=`$PYTHON -c \
