diff -Nurb o.ldns-0.70/Makefile.in n.ldns-0.70/Makefile.in
--- o.ldns-0.70/Makefile.in	2005-07-28 16:11:59.000000000 +0200
+++ n.ldns-0.70/Makefile.in	2005-08-13 20:08:04.395905096 +0200
@@ -8,6 +8,9 @@
 exec_prefix 	= @exec_prefix@
 bindir 		= @bindir@
 mandir 		= @mandir@
+includedir	= @includedir@
+libdir		= @libdir@
+DESTDIR		= 
 
 CC 		= @CC@
 CPPFLAGS	= @CPPFLAGS@ @DEFS@ -I. -I$(srcdir)
@@ -74,46 +77,47 @@
 	
 lib:		$(LIBDNS_OBJECTS) b64_pton$U.o b64_ntop$U.o
 		$(LINK_LIB) -o libldns.la $(LIBDNS_LOBJECTS) b64_pton$U.lo b64_ntop$U.lo \
-		-rpath $(prefix)/lib
+		-rpath $(libdir)
 
 doc:		doc/function_manpages 
 		doxygen libdns.doxygen
 		cat ldns/*.h | doc/doxyparse.pl -m doc/function_manpages 2>&1 | \
 		grep -v ^doxygen | grep -v ^cat  > doc/ldns_manpages
 
-install:	install-h install-lib
+install:	install-h install-lib install-doc
 
 uninstall:	uninstall-doc uninstall-h uninstall-lib
 
 destclean:	uninstall
 
 install-doc:	doc
-		${INSTALL} -d $(mandir)/man3
-		mv doc/man/man3/* $(mandir)/man3/
+		${INSTALL} -d $(DESTDIR)$(mandir)/man3
+		mv doc/man/man3/* $(DESTDIR)$(mandir)/man3/
 
 uninstall-doc:	doc
 		for i in `cat doc/ldns_manpages`; do \
-			rm -f $(mandir)/man3/$$i.3 ; done
-		[ ! -d $(mandir)/man3 ] || rmdir $(mandir)/man3
+			rm -f $(DESTDIR)$(mandir)/man3/$$i.3 ; done
+		[ ! -d $(DESTDIR)$(mandir)/man3 ] || rmdir $(DESTDIR)$(mandir)/man3
 		exit 0
 
 install-h:	lib
-		$(INSTALL) -m 755 -d $(prefix)/include/ldns
+		$(INSTALL) -m 755 -d $(DESTDIR)$(includedir)/ldns
+		$(INSTALL) -m 755 -d $(DESTDIR)$(libdir)
 		for i in $(LIBDNS_HEADERS); do \
-			MVPROG=/bin/cp $(INSTALL) -m 644 $$i $(prefix)/include/ldns/; done
+			MVPROG=/bin/cp $(INSTALL) -m 644 $$i $(DESTDIR)$(includedir)/ldns/; done
 
 uninstall-h:
 		for i in $(LIBDNS_HEADERS); do \
-			rm -f $(prefix)/include/$$i; done
-		[ ! -d $(prefix)/include/ldns ] || rmdir $(prefix)/include/ldns
+			rm -f  $(DESTDIR)$(includedir)/$$i; done
+		[ ! -d $(DESTDIR)$(includedir)/ldns ] || rmdir $(DESTDIR)$(includedir)/ldns
 		exit 0
 	
 install-lib:	lib
-		$(LIBTOOL) --mode=install cp libldns.la $(prefix)/lib
-		$(LIBTOOL) --mode=finish $(prefix)/lib
+		$(LIBTOOL) --mode=install cp libldns.la $(DESTDIR)$(libdir)
+		$(LIBTOOL) --mode=finish $(DESTDIR)$(libdir)
 
 uninstall-lib:	
-		$(LIBTOOL) --mode=uninstall rm $(prefix)/lib/libldns.la
+		$(LIBTOOL) --mode=uninstall rm  $(DESTDIR)$(libdir)/libldns.la
 
 snapshot:	realclean
 		(rm -rf ../ldns-snap-$(DATE)/)
@@ -169,10 +173,10 @@
 	rm -f doc/ldns_manpages
 
 libclean:
-	$(LIBTOOL) --mode clean rm -f libldns.la
-	$(LIBTOOL) --mode clean rm -f libldns.a
-	$(LIBTOOL) --mode clean rm -f libldns.so
-	$(LIBTOOL) --mode clean rm -f libldns.so.*
+	$(LIBTOOL) --mode=clean rm -f libldns.la
+	$(LIBTOOL) --mode=clean rm -f libldns.a
+	$(LIBTOOL) --mode=clean rm -f libldns.so
+	$(LIBTOOL) --mode=clean rm -f libldns.so.*
 	rm -rf *.lo
 	rm -rf .libs
 	rm -rf libtool
