--- ldns-1.9.0/ax_python_devel.m4.orig	2025-12-04 16:45:51.000000000 +0100
+++ ldns-1.9.0/ax_python_devel.m4	2026-06-07 14:01:23.916771189 +0200
@@ -312,7 +312,7 @@ print(sitedir)"`
 		else
 			# distutils.sysconfig way
 			PYTHON_SITE_PKG=`$PYTHON -c "$IMPORT_SYSCONFIG; \
-				print (sysconfig.get_python_lib(0,0));"`
+				print (sysconfig.get_python_lib(0,0,prefix='/usr'));"`
 		fi
 	fi
 	AC_MSG_RESULT([$PYTHON_SITE_PKG])
@@ -341,7 +341,7 @@ print(sitedir)"`
 		else
 			# distutils.sysconfig way
 			PYTHON_PLATFORM_SITE_PKG=`$PYTHON -c "$IMPORT_SYSCONFIG; \
-				print (sysconfig.get_python_lib(1,0));"`
+				print (sysconfig.get_python_lib(1,0,prefix='/usr'));"`
 		fi
 	fi
 	AC_MSG_RESULT([$PYTHON_PLATFORM_SITE_PKG])
