--- ldns-1.6.17/ax_python_devel.m4~	2014-01-10 22:04:41.000000000 +0100
+++ ldns-1.6.17/ax_python_devel.m4	2016-05-05 19:55:38.599100177 +0200
@@ -253,7 +253,7 @@
 	AC_MSG_CHECKING([for Python site-packages path])
 	if test -z "$PYTHON_SITE_PKG"; then
 		PYTHON_SITE_PKG=`$PYTHON -c "import distutils.sysconfig; \
-			print (distutils.sysconfig.get_python_lib(1,0));"`
+			print (distutils.sysconfig.get_python_lib(1,0,prefix='/usr'));"`
 	fi
 	AC_MSG_RESULT([$PYTHON_SITE_PKG])
 	AC_SUBST([PYTHON_SITE_PKG])
